src/thirdparty/devtools-frontend/index.js

```diff
@@ -3232,7 +3185,7 @@ class HeapSnapshot {
         let j = 0;
         const l = baseIds.length;
         const m = indexes.length;
-        const diff = new Diff(aggregate.name);
+        const diff = new Diff(aggregate ? aggregate.name : baseAggregate.name);
```

Comment:
This looks like it may be designed exactly to mitigate #119.

Related: https://github.com/ChromeDevTools/devtools-frontend/commit/204e83192cc0720501e89c3bdf6f35a86abb157d